### ISSUE TITLE
Emotionet and Batchnormalization

### ISSUE THREAD

**Issue by kannadaraj** [CookiePPP/cookietts#36]
Hi,

   I was curious why haven't you used Batch normalization in emotion net for while traversing through he convolution layers.? Is there any specific reason. Was just curious because that might bring in more stability while training right?

Thanks

**Comment by CookiePPP** [comment]
I don't remember that long ago very well but it was likely me just testing an idea so I put minimal effort into the module code.

**Comment by kannadaraj** [comment]
Thanks for the reply. I get it. Let me make try out few extra things in emotionnet and will tell you how it turned out.

**Comment by kannadaraj** [comment]
Just asking, with the present emotionnet were you able to perform good style transfer? Thanks for any comments. :)

**Comment by CookiePPP** [comment]
I don't remember almost anything about EmotionNet, so I don't think it worked out.

**Comment by kannadaraj** [comment]
Thanks.
